deleteEarn.py

```diff
@@ -0,0 +1,21 @@
+# Time Complexity : O(N log N) where N = len(nums)
```

Comment:
Is this because of collections.Counter() method? Because I have read otherwise: https://stackoverflow.com/questions/42461840/what-is-the-time-complexity-of-collections-counter-in-python

Reply:
Got it, updated the code. Thank you Sowmya!